examples/cli/src/main/java/io/unitycatalog/cli/SchemaCli.java

```diff
@@ -65,11 +66,20 @@ private static String getSchema(SchemasApi schemasApi, JSONObject json) throws J
     }
 
     private static String updateSchema(SchemasApi schemasApi, JSONObject json) throws JsonProcessingException, ApiException {
-        UpdateSchema updateSchema = new UpdateSchema()
-                .newName(json.optString(CliParams.NEW_NAME.getServerParam()))
-                .comment(json.optString(CliParams.COMMENT.getServerParam()));
-        return objectWriter.writeValueAsString(schemasApi
-                .updateSchema(json.getString(CliParams.FULL_NAME.getServerParam()), updateSchema));
+        String schemaFullName = json.getString(CliParams.FULL_NAME.getServerParam());
+        json.remove(CliParams.FULL_NAME.getServerParam());
+        if (json.length() == 0) {
+            List<CliParams> optionalParams = CliUtils.cliOptions.get(CliUtils.SCHEMA)
+                .get(CliUtils.UPDATE)
+                .getOptionalParams();
+            String errorMessage = "No parameters to update, please provide one of:";
+            for (CliParams param : optionalParams) {
+                errorMessage += "\n  --" + param.val();
+            }
+            throw new CliException(errorMessage);
```

Comment:
so, is the current model that the CLi throws an error on the client side but the server if still somehow receives such a request (from other clients) we will do a no-op?

Reply:
Pretty much except that the CLI does not throw the exception in `main`; I reverted that change we talked about in the catalog PR back to the original way that the CLI `main` handled `CliException`s [in this discussion](https://github.com/unitycatalog/unitycatalog/pull/158#discussion_r1670910208).

So in short:
- if the server gets an empty update, result is a no-op with success,
- if the CLI gets an empty update, the `updateXyz` handler does not call the server and instead throws a `CliException` with a help message, and the CLI `main` prints the message but result is still a success.